### ISSUE TITLE
Share record proto definition with other implementations?

### ISSUE THREAD

**Issue by anacrolix** [libp2p/js-libp2p-record#12]
The proto file for Record is a copy of the one in go-libp2p-record. Is it possible/worth sharing the definitions?

**Comment by vasco-santos** [comment]
I totally agree that we should share this definition, as well as others like IPNS record. however, I do not know of a good solution for it.

**Comment by anacrolix** [comment]
Can https://github.com/libp2p/js-libp2p-record/blob/master/src/record.proto.js and similar definitions be pulled from another repo? Is there something more involved than that?
